java-checks/src/test/java/org/sonar/java/checks/spring/SpringBeansShouldBeAccessibleCheckTest.java

```diff
@@ -93,6 +93,22 @@ void testComponentScan() {
       .verifyNoIssues();
   }
 
+  @Test
+  void testComponentScanWithBasePackageClasses() {
+    final String testFolder = BASE_PATH + "componentScan/";
+    List<String> files = Arrays.asList(
```

Comment:
For this test and the others, we can probably replace `Arrays.asList` with `List.of` since we do not need the lists to be mutable